CP.md

```diff
@@ -41,7 +42,7 @@ The following revisions have been made:
 | September 20, 2018 | Define Certificate Problem Report in Section 1.6.1. Update Section 3.2.2.4 "Validation of Domain Authorization or Control" to match latest BRs but omit methods we do not use. Add additional revocation reason to Section 4.9.1.1 for compliance with upcoming BR revision. Minor updates to Sections 4.9.3 and 4.9.5. | 2.2 |
 | July 3, 2019 | Remove IP address validation information which is not applicable. Update sections 4.9.1.1, 4.9.1.2, and 4.9.5 to match current BRs. Other minor updates (e.g. capitalization). | 2.3 |
 | January 21, 2020 | Make structure more exactly match RFC 3647 recommendation. Audit use of phrase No Stipulation and eliminate blank sections. Add policy information for IP address validation. | 2.4 |
-| X Y, Z | Update sections 3.2.2.4, 3.2.2.5, 3.2.2.6, 4.2.1, 4.9.10, 7.1.2, and 7.1.3 regarding validation methods, OCSP, certificate profiles, and cryptographic algorithms, to match Baseline Requirements v1.7.2. | 2.5 |
+| September 30, 2020 | Update sections 3.2.2.4, 3.2.2.5, 3.2.2.6, 4.2.1, 4.9.10, 7.1.2, and 7.1.3 regarding validation methods, OCSP, certificate profiles, and cryptographic algorithms, to match Baseline Requirements v1.7.2. | 2.5 |
```

Comment:
We don't add a date here until PMA has approved the updates and the release date is known. Please return to X Y, Z.

Please do add an entry at the beginning of the current changes listed here that says something like "List new ISRG Root X2."

Reply:
Gotcha, done.